openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/explore/explore.java

```diff
@@ -0,0 +1,363 @@
+package org.openmetadata.catalog.selenium.pages.explore;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.*;
+import org.openmetadata.catalog.selenium.objectRepository.*;
+import org.openmetadata.catalog.selenium.pages.tableDetails.TableDetailsPageTest;
+import org.openmetadata.catalog.selenium.properties.Property;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+public class explore extends TableDetailsPageTest {
+  WebDriver webDriver;
+  static String url = Property.getInstance().getURL();
+  static Actions actions;
+  static WebDriverWait wait;
+  Integer waitTime = Property.getInstance().getSleepTime();
+  String tableName = "dim_address";
+
+  @BeforeEach
+  public void openMetadataWindow() {
+    System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/linux/chromedriver");
+    ChromeOptions options = new ChromeOptions();
+    options.addArguments("--headless");
+    options.addArguments("--window-size=1280,800");
+    webDriver = new ChromeDriver();
+    actions = new Actions(webDriver);
+    wait = new WebDriverWait(webDriver, Duration.ofSeconds(30));
+    webDriver.manage().window().maximize();
+    webDriver.get(url);
+  }
+
+  @Test
+  @Order(1)
+  public void openExplorePage() throws InterruptedException {
+    explorePage explorePage = new explorePage(webDriver);
+    myDataPage myDataPage = new myDataPage(webDriver);
+    myDataPage.closeWhatsNew().click();
```

Comment:
Use click operation from Events.java.

Comment applies to other instances of this in this PR